test/distributed/org/apache/cassandra/distributed/test/accord/AccordTestBase.java

```diff
@@ -183,6 +196,27 @@ public static void install(ClassLoader classLoader, Integer num)
             return map;
         }
     }
-    
+
+    public static class BBAccordCoordinateCountHelper
+    {
+        static AtomicInteger count = new AtomicInteger();
+        static void install(ClassLoader cl, int nodeNumber)
+        {
+            if (nodeNumber != 1)
+                return;
+            new ByteBuddy().rebase(AccordService.class)
+                           .method(named("coordinate").and(takesArguments(2)))
+                           .intercept(MethodDelegation.to(BBAccordCoordinateCountHelper.class))
+                           .make()
+                           .load(cl, ClassLoadingStrategy.Default.INJECTION);
+        }
+
+        public static TxnData coordinate(Txn txn, @SuperCall Callable<TxnData> actual) throws Exception
+        {
+            count.incrementAndGet();
+            return actual.call();
+        }
+    }
```

Comment:
I have as one of the TODO items in [CASSANDRA-18107](https://issues.apache.org/jira/browse/CASSANDRA-18107) latency metrics recording in `AccordService#coordinate()`, probably as a "Transaction" item in `ClientRequestsMetricsHolder`. Maybe we could just throw that in here, avoid BB, and kill two birds w/ one stone?

Reply:
Seems reasonable. Do we want separate metrics for transactions that are write only since they generally have a very different latency profile?

Reply:
This looks more complicated than just latency since it's counting a bunch of different things. Abort, timeout, unavailable, mutation size. I think we need to think a little harder about metrics for Accord transactions.

Reply:
Agreed, although we already handle similar concerns in `CASClientWriteRequestMetrics` -> `CASClientRequestMetrics` -> `ClientRequestMetrics`, where we distinguish between those things.

Reply:
I'm find if you want to defer to [CASSANDRA-18107](https://issues.apache.org/jira/browse/CASSANDRA-18107)